### PR TITLE
fix: update references for avro and openapi schema

### PR DESCRIPTION
**Pull request by derberg**
autocompletion in IDEs is affected

Screenshot 2023-05-24 at 09 19 10

some more rewriting in refs in the dedicated JSON Schema was needed

- `schemas` content is generated, you can skip review of this one
- added `test/fixtures/asyncapi.yml` to make it easier long term to test schemas on local, readme also updated
- main magic changes in `index.js`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,7 +173,24 @@ The manual process of creating a new version is to:
...
+
+Whenever you make changes in AsyncAPI JSON Schema, you should always manually verify that the schema is still supported by [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) and that it will be able to fetch and dereference it.
+
+- Make sure you have VSCode and YAML extension
+- Once you make changes in schemas, bundle them with `npm run bundle`
+- Edit [this sample AsyncAPI document](./test/fixtures/asyncapi.yml). Add a new field, hover over some property to see if tooltip still shows up. This sample document contains the following line that assures YAML uses your local schema and not the one from SchemaStore. Make sure it points to the schema bundle that you generated and that contains your changes:
+   ```yaml
+   # yaml-language-server: $schema=/Users/wookiee/sources/asyncapi-node/schemas/2.6.0-without-$id.json
```
Comment (dalelane) [inline README.md:191]: "wookiee"?  :-)

File: test/fixtures/asyncapi.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# yaml-language-server: $schema=/Users/wookiee/sources/asyncapi-node/schemas/2.6.0-without-$id.json
```
Comment (dalelane) [inline test/fixtures/asyncapi.yml:1]: should we make this a more generic placeholder?

e.g. `/YOUR-HOME-DIRECTORY/sources/...`

**Review by dalelane — APPROVED** [review]
one stylistic question, but otherwise looks good 👍

File: README.md (hunk 1 above)
Comment (derberg) [reply README.md:191]: hehe, yeah, that's what my computer is called, what I can do, I'm a fan of star wars and Chewbacca is my favourite 😆

**Review by derberg — COMMENTED** [review]
@dalelane removed wookiee 😄 please have another look

**Review by fmvilas — APPROVED** [review]
LGTM 👍

**Pull request merged**